doc/faq.rst

```diff
@@ -181,86 +181,59 @@ Using Spyder
 
-   .. image:: /images/faq/faq-change-environment.png
-      :alt: Change default environment in Preferences option in status bar
-
-   This will open the :guilabel:`Preferences` dialog in the :guilabel:`Python interpreter` section.
+   To do so, open the :guilabel:`Python interpreter` section of Spyder's preferences (:menuselection:`Tools --> Preferences`, or :menuselection:`Spyder --> Preferences` on macOS).
    Here, select the option :guilabel:`Use the following Python interpreter`, and use the dropdown below to select your preferred environment.
-   If its not listed, use the text box or the :guilabel:`Select file` button to enter the path to the Python interepreter you want to use.
-   See the :doc:`/panes/ipythonconsole` for more information.
+   If it's not listed, see :ref:`the note below <faq-env-not-listed>`.
 
    .. image:: /images/faq/faq-python-interpreter.png
       :alt: Preferences showing changing Python interpreter
 
-   Click :guilabel:`Restart kernel` in the :guilabel:`Consoles` menu for this change to take effect.
+   .. _faq-env-not-listed:
+
+   .. note::
+
+      If you installed Miniconda (or another Conda-based distribution) to a non-default path, or are using a virtual environment managed by a tool other than ``pyenv``, your environments likely won't be listed.
+
+      Instead, use the text box or the :guilabel:`Select file` button to enter the path to the Python interpreter you want to use.
+      You can find this path by activating the venv or Conda env you want to use in your terminal/Anaconda Prompt, and running the shell command:
+
+      .. code-block:: shell
+
+         python -c "import sys; print(sys.executable)"
+
+   Finally, click :guilabel:`Restart kernel` in the :guilabel:`Consoles` menu for this change to take effect.
+   If ``spyder-kernels`` is not already installed, the :doc:`/panes/ipythonconsole` will display instructions on how to install the right version.
+   Execute the given command in your terminal/Anaconda prompt with the environment activated, and finally restart the kernel once more.
 
 
 .. _using-packages-installer:
 
 .. dropdown:: Q: How do I install Python packages to use within Spyder if I downloaded Spyder from the standalone installers?
    :container: + dropdown-id-using-packages-installer
 
-   Watch our video on using additional modules or follow the instructions below it.
+   Watch our video on using additional packages or follow the instructions below.
 
    .. youtube:: i7Njb3xO4Fw
       :height: 360
       :width: 640
       :align: left
       :start: 306
 
-   If you want to use other modules in Spyder that don't come with our installer, you need to install `Miniconda`_ (**only if you don't have Anaconda or Miniconda yet!**).
-   For Spyder to recognize it, the installation should be done in one of the following default paths:
-
-   .. table::
```

Comment:
I think keeping this table is important so that users know where they can install Anaconda/Miniconda for Spyder to detect their environments.

Why do think we should remove it?

Reply:
I explain this in the OP:

> * Replaces the long, unwieldy and hard to update table of default paths (which will get >2x as long when spyder-ide/spyder#20498 is merged, and doesn't even include Linux) with a much more concise, actionable and maintainble mention of installing a conda-based distribution to the installer's default directory.

To answer you more specifically, users don't need to do anything for this, they just have to accept the default install path, as the text below explains. This table just lists the default install paths for each platform (Windows/Mac), distribution (Anaconda, Miniconda, Miniforge, etc), Anaconda version (older versions have a different default), Python version (Python 2, Python 3) and installation type (per-user, system), so it is effectively redundant (as only one of them will apply to each user).

Especially in the context of an FAQ aimed at newer users, it's much simpler to just tell them to accept the default install path, then confuse them with a huge table of complex paths and have them try to laboriously try to compare their default path with these. Already, it takes up a full screen's worth of space and over doubles the total length of the FAQ, and in order to not be out of date, we'd need to update it with the Miniforge and Mambaforge variants of each path too (plus maybe some additional variations, if I remember right) which would >=double it's length again, taking up >=two full screens of space. Plus, if we add more paths in the future, it gets even longer and harder to use and maintain.

Reply:
> Especially in the context of an FAQ aimed at newer users, it's much simpler to just tell them to accept the default install path, then confuse them with a huge table of complex paths and have them try to laboriously try to compare their default path with these

Ok, this is a very good point. Then I agree with removing the table.